config/xml_schemas/config_batch.xsd

```diff
@@ -78,6 +79,9 @@
   <!-- walltime_format: time format expected by batch system for the wall clock time field -->
         <xs:element minOccurs="0" ref="walltime_format"/>
 
+  <!-- batch_jobid_env_var: environment variable set by the batch system indicating the
+       job id for the submitted job -->
+        <xs:element minOccurs="0" ref="batch_jobid_env_var"/>
```

Comment:
you have minOccurs = 0 but it looks like this is now a required field?

Reply:
I check and verify it exists in `EnvBatch.is_submission_job`, so it shouldn't be not required

Reply:
okay - what about the case in which you are not using a batch system at all?

Reply:
TBC, I do prefer the idea of adding a command line argument to case.st_archive; no new xml and doesn't have this issue

Reply:
@jedwards4b Another reason to prefer the command line option - it solves the issue of RESUBMIT being set by the user during a run in the other PR I have. If it's okay, I'll just add that method to that branch, and close this PR?